llm/server.go

```diff
@@ -640,20 +640,20 @@ root   ::= object
 value  ::= object | array | string | number | ("true" | "false" | "null") ws
 object ::=
   "{" ws (
-            string ":" ws value
+            ws string ":" ws value
     ("," ws string ":" ws value)*
-  )? "}" ws
+  )? ws "}" 
```

Comment:
For this and line 650 below, this change doesn't actually remove the trailing "trailing" ws, correct? As this is handled on line 640? (below)

```
value ::= object | array | string | number | ("true" | "false" | "null") ws
```